### PR TITLE
4th Project (solo) Apple page from 2014

### PR DESCRIPTION
**Pull request by lucasmazo32**
Apple Web Page:

Using gradients in the nav bar.
Organize all element with flex or float.
Using a tags to use them a as a link.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,281 @@
+/* font */
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  display: grid;
```
Comment (pelzolga123) [inline style.css]: You should use flex or float here.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Some changes are required.

**Comment by pelzolga123** [comment]
## Hi Lucas!  
### Your project looks :+1: great!
#### But some changes are required, before project can be approved:
* [ ] Pull request description should have info about this project, not a request for review. [Link to Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
* [ ] All elements should be aligned with float OR flex. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/float_or_flex_requirement.png)

Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!


### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Layout needs to be corrected.

**Comment by pelzolga123** [comment]
### Good work adding PR descriptive message :+1: 
Some fixes are still needed:
* [x] Layout is not displaying well : 
image
[Optional]:
 * Also, why do you use JQuery for this project? It is not required. You can copy images from original website and add them inside your project.
* You mentioned in your README file, that you don't know some things, it looks not very professional, I suggest for you to remove it.

### Please kindly fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts).  *Happy coding*

**Review by imhta — APPROVED** [review]
# Project approved 🎉 

### Hello  @lucasmazo32  , 👋 You have done a good job in this project 👏 



----
### Happy coding 😄 
----
**Pon Muthu Selvam N**
__nponmuthuselvam@gmail.com__

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Terrific job implementing some of the changes requested by the previous reviewer! :rocket: 
There are a few more things to do before you can merge :+1: .

#### Design
Your layout has a bug because it currently has a bit of horizontal scroll when it shouldn't. Please look into this and fix

#### Readme File
In addition to what was previously mentioned about your Readme File, the file is supposed to make potential project viewers as excited as possible about the prospects of viewing your project and not deter them or lower their interests before they've had a chance to judge the work for themselves. Please consider an update to this file and use [this](https://github.com/Oluwadamilareolusakin/weather-lord/blob/master/README.md) as an example of what to include in yours.
screenshot

#### Summary
You have done a splendid job @lucasmazo32 you just need to make those few changes and you're great to go :smile:

**Pull request merged**